org-code-javabuilder/theater/src/main/java/org/code/theater/Theater.java

```diff
@@ -0,0 +1,181 @@
+package org.code.theater;
+
+public enum Instrument {
+   PIANO, BASS
+}
+
+public static class Theater {
+   /**
+    * Plays the array of samples provided.
+    * 
+    * @param sound an array of samples to play.
+    */
+   public static void playSound(double[] sound) {
+   }
+
+   /**
+    * Plays the sound referenced by the file name.
+    * 
+    * @param filename an array of samples to play.
+    * @throws SoundException if the file can't be found in the project.
+    */
+   public static void playSound(String filename) throws SoundException {
+   }
+
+   /**
+    * Plays a note with the selected instrument.
+    * 
+    * @param instrument the instrument to play.
+    * @param note       the note to play. 60 represents middle C on a piano.
+    * @param seconds    length of the note. Implementer's note: Behind the scenes,
+    *                   this should just be implemented using an array of loopable
+    *                   sounds (Mike can generate these).
+    */
+   public static void playNote(Instrument instrument, int note, double seconds) {
+   }
+
+   /**
+    * Wait the provided number of seconds before performing the next draw or play
+    * command.
+    * 
+    * @param seconds The number of seconds to wait. This can be a fraction of a
+    *                second, but the smallest value can be .1 seconds.
+    */
+   public static void wait(double seconds) {
+   }
+
+   /**
+    * Clear everything, starting from the beginning and emptying the canvas.
+    */
+   public static void reset() {
+   }
+
+   /**
+    * Draw an image on the canvas at the given location, expanded or shunk to fit
+    * the width and height provided
+    * 
+    * @param file     the name of the file in the asset manager
+    * @param x        the left side of the image in the canvas
+    * @param y        the top of the image in the canvas
+    * @param width    the width to draw the image on the canvas
+    * @param height   the height to draw the image on the canvas
+    * @param rotation the amount to rotate the image
+    */
+   public static void drawImage(String filename, double x, double y, double width, double height, double rotation)
+         throws ImageException {
```

Comment:
when does it throw an `ImageException`?

Reply:
Oy, yea I think this needs to be a `FileNotFoundException` too.